src/main/java/hudson/plugins/parameterizedtrigger/BuildTriggerConfig.java

```diff
@@ -569,7 +566,7 @@ protected Future schedule(@Nonnull AbstractBuild<?, ?> build, @Nonnull final Job
                 return null;
             }
                    
-            return parameterizedJobMixIn.scheduleBuild2(quietPeriod, queueActions.toArray(new Action[queueActions.size()]));
+            return parameterizedJobMixIn.scheduleBuild2(quietPeriod, queueActions.toArray(new Action[0]));
```

Comment:
Was there a reason to always choose 0 here rather than the more dynamic size()?

Reply:
Using `0` is the simpler and possibly even faster way - see the commit's f03d4b4a7729e8815194a0f110f8e4ae8c5eb3fd comment.